src/Config.ts

```diff
@@ -95,10 +98,10 @@ export class FacilitatorConfig {
 
   /**
    * This reads facilitator config from the json file and creates FacilitatorConfig object.
-   * @param {string} chain Auxiliary chain id.
-   * @returns {FacilitatorConfig} Facilitator config object.
+   * @param chain Auxiliary chain id.
+   * @returns Facilitator config object.
    */
-  public static from(chain: string): FacilitatorConfig {
+  public static fromChain(chain: string): FacilitatorConfig {
```

Comment:
I assume, the function should call `fromFile` function after constructing the config file path.

Reply:
As fromFile throws exception now, I would not like to call `fromFile` method. It will help to create facilitator-config file at default path directory if it doesn't exists.
Also, It is consistent with mosaic-chains.